src/utils/helpers.ts

```diff
@@ -0,0 +1,40 @@
+import { createComment } from "./create-comment.js";
+
+const profilePicture =
+  "https://cdn.pixabay.com/photo/2021/05/20/11/58/beauty-6268460_1280.jpg";
+
+export const likeComment = (
+  likesInfoElement: HTMLElement,
+  commentSvgElement: HTMLImageElement
+) => {
+  if (likesInfoElement.innerText === "like") {
+    likesInfoElement.innerText = "1 like";
+    commentSvgElement.src = "../public/assets/red-heart.svg";
+  } else {
+    likesInfoElement.innerText = "like";
+    commentSvgElement.src = "../public/assets/heart.svg";
+  }
+};
+
+export const addEventListenerWhenPageLoaded = () => {
```

Comment:
If you add an event Listner like this then you should also remove the event listener once its no longer needed,
otherwise it will use unwanted resources.

Instead of using an event you call this function once "on load"

Reply:
add { once: true }as a second argument